src/Tickets/Admin/All_Tickets/Provider.php

```diff
@@ -51,5 +51,9 @@ protected function register_hooks() {
 	 * @since TBD
 	 */
 	protected function register_assets() {
+		$assets = new Assets( $this->container );
+		$assets->register();
+
+		$this->container->singleton( Assets::class, $assets );
```

Comment:
```suggestion
		$this->container->register( Assets::class );
```
Same result since the container's register method will create a sisngleton and jut call the register method of the registered interface immediately.